include/zephyr/arch/arm/cortex_m/exception.h

```diff
@@ -119,6 +119,15 @@ struct arch_esf {
 
 extern uint32_t z_arm_coredump_fault_sp;
 
+/* Size of the exception frame pushed by hardware, in bytes. The stack pointer at the time of the
+ * exception is the ESF base address plus this size.
+ */
+#if defined(CONFIG_EXTRA_EXCEPTION_INFO)
+#define Z_ARCH_HW_ESF_SIZE offsetof(arch_esf, extra_info)
```

Comment:
This might need to be name Z_ARM_* rather than Z_ARCH as this is not really a zephyr Arch thing but a very arm support specific define.

Reply:
Renamed to `Z_ARM_MACHINE_ESF_SIZE`.